itest/universe_test.go

```diff
@@ -122,6 +124,44 @@ func testUniverseSync(t *harnessTest) {
 	)
 	assertUniverseKeysEqual(t.t, uniIDs, t.tapd, bob)
 	assertUniverseLeavesEqual(t.t, uniIDs, t.tapd, bob)
+
+	// We should also be able to fetch an asset from Bob's Universe, and
+	// query for that asset with the compressed script key.
+	firstAssetID := rpcSimpleAssets[0].AssetGenesis.AssetId
+	firstScriptKey := hex.EncodeToString(rpcSimpleAssets[0].ScriptKey)
+	firstOutpoint, err := tap.UnmarshalOutpoint(
+		rpcSimpleAssets[0].ChainAnchor.AnchorOutpoint,
+	)
+	require.NoError(t.t, err)
+	require.Len(t.t, firstScriptKey, btcec.PubKeyBytesLenCompressed*2)
```

Comment:
Why is that true?

Reply:
Which part, fetching the asset or the key length?

We synced the minted assets to Bob's universe earlier, and we always display the key in compressed form, so as a hex-encoded string it should always be 66 characters.

Reply:
Length makes sense now, thanks :+1: